### ISSUE TITLE
ProblemMatcher - owner

### ISSUE THREAD

**Issue by Springrbua** [microsoft/vscode-typescript-tslint-plugin#95]
When I run tslint as a task, using the `$tslint5` `ProblemMatcher`, the problems will be shown twice, as soon as I open an affected file. When I fix the problem, one of them will disappear, but the one from the task will still remain.
The same task worked well with the now [deprecated TSLint Plugin](https://marketplace.visualstudio.com/items?itemName=eg2.tslint).
My guess is, that since this plugin is running tslint as a tsserver-plugin, the owner of the problems is not `tslint` anymore.
So the [`owner`-property](https://github.com/microsoft/vscode-typescript-tslint-plugin/blob/master/package.json#L87) of the `ProblemMatcher`s should probably be changed to the `owner` of `tsserver`.

**Comment by Springrbua** [comment]
I was looking for the owner name of the `tsserver` and found that it is `typescript`. I then modified my `lint` task to use that owner and it worked as expected.
However, there is one issue with that:
If the tsserver is not started yet, all warnings will be cleared as soon as tsserver starts (for example when opening the first *.ts file).  
Therefore I searched for the `$tsc` `problemMatcher` and found that it only [applies to closed documents](https://github.com/microsoft/vscode/blob/cc799ebf60d4d01e75dde19ce92f46b7711c2b71/extensions/typescript-language-features/package.json#L743).
This seems to be a good solution for the tslint problem matcher too.
